### ISSUE TITLE
Reaching out to other Ansible ACI developers

### ISSUE THREAD

**Issue by dagwieers** [datacenter/aci-ansible#109]
There are quite some ACI Ansible developers/users on Github and blogs on the Interweb, and since we have 2 weeks left before these modules/interfaces are going to be set in stone in Ansible v2.4, getting feedback from known experts in this field is both essential and urgent.

So I am reaching out to: @jedelman8 @joelwking @theothercpaggen @cgascoig @thinkpradhap @mamullen13316 @jjgrinwis @rbenigno @dstamen @ericchou1 @ryantischer @cpaggen 

Please provide feedback on these Cisco ACI modules that will ship in Ansible v2.4. The freeze is on August 29, so insights from your area of expertise are very much appreciated !

**Comment by joelwking** [comment]
I see an issue with the changed flag not set correctly with the  aci_rest module. I assume you would like me to open a new issue here to document?

**Comment by dagwieers** [comment]
@joelwking I don't know if we can implement idempotency with aci_rest (in all its versaility), if you have opinions about that, yes please :-)

**Comment by joelwking** [comment]
Hope this helps.
https://github.com/datacenter/aci-ansible/issues/111

**Comment by joelwking** [comment]
Test results of `aci_rest` located at https://gist.github.com/joelwking/5f2609f704dba1adc7ac41593c74b291

**Comment by dagwieers** [comment]
@joelwking Thanks for that. We'll have to look at improving the error-message, the reason it says "Authentication error" was because the exception is part of the authentication phase.

For the aci_rest idempotency, I have  the needed urllib code for adding parameters to the query string, but since that part of the code is under review by @jmcgill298 I haven't merged it yet.

**Comment by dagwieers** [comment]
So the Ansible v2.4 RC1 release is getting very close now. If anyone wanted to test-drive these ACI modules, now is the time. Once stable-2.4 is branched (expected 2017-09-06), it will be harder to get changes into Ansible v2.4.

**Comment by dagwieers** [comment]
With the v2.5 deadlines coming up, we would like everybody interested in Ansible + ACI to know that we have a new action list and a large set of new modules to create. So [the ACI action plan has been updated](https://github.com/ansible/community/wiki/Network:-ACI) and we will soon have [new ACI development documentation](https://github.com/ansible/community/wiki/Network:-ACI-Documentation).

Let us know if you are interested to help out, test-drive the newer modules or join the weekly meetings !
Feedback is always appreciated.

PS We decided to work in upstream first, so this repository will slowly turn into a placeholder with hopefully some Ansible + ACI walk-throughs.
